llvm/lib/CodeGen/SelectionDAG/LegalizeDAG.cpp

```diff
@@ -1377,6 +1378,19 @@ void SelectionDAGLegalize::LegalizeOp(SDNode *Node) {
   }
 }
 
+// Helper function that generates an MMO that considers the alignment of the
+// stack, and the size of the stack object
+MachineMemOperand *getStackAlignedMMO(SDValue &StackPtr, MachineFunction &MF) {
```

Comment:
Don't bother passing by ref - `SDValue StackPtr`

Reply:
Also this function should be `static`